internal/configs/config_bootloaders.go

```diff
@@ -83,7 +83,7 @@ func Set_KernelStub(isRoot bool) {
 
 	// Run and log, check for errors
 	common.ErrorCheck(command.ExecAndLogSudo(isRoot, true,
-		"kernelstub -a "+kernel_args,
+		"kernelstub -a "+fmt.Sprintf("\"%s\"", kernel_args),
```

Comment:
just out of curiosity, wouldn't `fmt.Sprintf("kernelstub -a \"%s\"", kernel_args),` be more readable, or is that just a preference thing?

anyways reinstalling my popos vm for testing.

Reply:
The good news though, it seems like everything else afterwards will execute fine :tada: